### PR TITLE
add io.neovim.nvim.tool.ripgrep

### PR DESCRIPTION
**Pull request by Lctrs**
ripgrep is a strongly recommended dependency of telescope.nvim, a popular plugin to - among other things - search strings inside files of current working directory.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Lctrs** [comment]
bot, build io.neovim.nvim.tool.ripgrep

File: io.neovim.nvim.tool.ripgrep.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+id: io.neovim.nvim.tool.ripgrep
+default-branch: stable
```
Comment (hfiguiere) [inline io.neovim.nvim.tool.ripgrep.yaml:2]:
```suggestion
```

File: io.neovim.nvim.tool.ripgrep.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          tag-query: ".tag_name"
+          timestamp-query: ".published_at"
+          is-main-source: true
+      - generated-sources.json
+  - name: appdata
+    buildsystem: simple
+    build-commands:
+      - install -Dm0644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
+    sources:
```
Comment (hfiguiere) [inline io.neovim.nvim.tool.ripgrep.yaml:42]:
```suggestion
      - install -Dm0644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
    sources:
      - type: git
        url: https://github.com/BurntSushi/ripgrep.git
        tag: 13.0.0
        commit: af6b6c543b224d348a8876f0c06245d9ea7929c5
        x-checker-data:
          type: json
          url: https://api.github.com/repos/BurntSushi/ripgrep/releases/latest
          tag-query: ".tag_name"
          timestamp-query: ".published_at"
          is-main-source: true
      - generated-sources.json
```

**Comment by Lctrs** [comment]
bot, build io.neovim.nvim.tool.ripgrep

**Comment by Lctrs** [comment]
bot, build io.neovim.nvim.tool.ripgrep

**Comment by Lctrs** [comment]
bot, build io.neovim.nvim.tool.ripgrep

**Comment by Lctrs** [comment]
@bbhtt Updated the PR. Is it good for you ?

**Pull request closed without merging**